ntoskrnl/ps/query.c

```diff
@@ -212,9 +212,13 @@ NtQueryInformationProcess(
             /* Get additional information, if needed */
             if (Extended)
             {
+                QuotaLimits.Flags |= (Process->Vm.Flags.MaximumWorkingSetHard ?
+                    QUOTA_LIMITS_HARDWS_MAX_ENABLE : QUOTA_LIMITS_HARDWS_MAX_DISABLE);
+                QuotaLimits.Flags |= (Process->Vm.Flags.MinimumWorkingSetHard ?
+                    QUOTA_LIMITS_HARDWS_MIN_ENABLE : QUOTA_LIMITS_HARDWS_MIN_DISABLE);
+
                 /* FIXME: Get the correct information */
                 //QuotaLimits.WorkingSetLimit = (SIZE_T)-1; // Not used on Win2k3, it is set to 0
-                QuotaLimits.Flags = QUOTA_LIMITS_HARDWS_MIN_DISABLE | QUOTA_LIMITS_HARDWS_MAX_DISABLE;
                 QuotaLimits.CpuRateLimit.RateData = 0;
```

Comment:
Note: This assignment is currently redundant after `RtlZeroMemory()`.

Reply:
It's not the correct information right now and only kept for reference, along with other values under FIXME. This will be changed later when it's implemented.

Reply:
I think it's fine to keep as a reminder.